nsxt/resource_nsxt_edge_transport_node.go

```diff
@@ -97,6 +97,12 @@ func resourceNsxtEdgeTransportNode() *schema.Resource {
 			"description":  getDescriptionSchema(),
 			"display_name": getDisplayNameSchema(),
 			"tag":          getTagsSchema(),
+			"node_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Unique Id of the fabric node",
+			},
```

Comment:
This is probably a bad idea, but I'll suggest it anyway.
As this resource is not generated, we do not have to fully respect the NSX API schema.

Perhaps we can move node_id in deployment_config and specify that it conflictsWith vm_deployment_config?